src/jdk.compiler/share/classes/com/sun/tools/javac/comp/Attr.java

```diff
@@ -5471,8 +5471,19 @@ void attribClass(ClassSymbol c) throws CompletionFailure {
                                                         .stream()
                                                         .anyMatch(d -> d.tsym == c);
                             if (!thisIsASuper) {
-                                log.error(TreeInfo.diagnosticPositionFor(subType.tsym, env.tree),
-                                        Errors.InvalidPermitsClause(Fragments.DoesntExtendSealed(subType)));
+                                KindName subtypingLeftKind;
```

Comment:
I wonder, would: `KindName.kindName(subType.tsym)` work?

Reply:
Indeed and I updated the PR. Thanks @lahodaj